src/openforms/formio/components/custom.py

```diff
@@ -712,6 +712,7 @@ def validate_list(self, partners):
             source=FormVariableSources.user_defined,
             prefill_plugin=FM_PLUGIN_IDENTIFIER,
             prefill_options__mutable_data_form_variable=component_key,
+            form=submission.form.id,
```

Comment:
```suggestion
            form=submission.form,
```
it will already be loaded, so there's no performance gain here, and even if you want to optimize for performance, then you'd use `submission.form_id` because the `.id` makes django do a query for the form instance in this form.